model/cat/indices/cat_indicies/structures.smithy

```diff
@@ -35,8 +35,8 @@ structure GetCatIndicesInput {
 
 structure GetCatIndicesOutput {
 
-   @required
-   body:Document
+    @httpPayload
+    content: Document,
```

Comment:
[NIT] is the comma at the end needed, similar for output. check for others as well. Lot of last parameters in structures have unnecessary commas in ends